### PR TITLE
Fix profile.py

### PR DESCRIPTION
**Pull request by jedi22**
Addressing: #2892 
This fixes a lot of the issues but still doesn't address // when it's in a query because we can't just ignore it (which is what we now do with URLs in descriptions.

@poppyseedplehzr Any regex fu ideas?

### CODE REVIEW

**Comment by jedi22** [comment]
One thing we could do is choose to not support C++ style comments in packs files (there aren't any there right now)

**Comment by jedi22** [comment]
If we remove support for things like
```
{	// A comment
	"some_json" : "some_value" // Another comment
}
```
Which as far as I can tell we don't use (we will still allow single line comments on their own line) I think we can fix the parsing

File: tools/tests/utils.py [hunk 1]
```diff
@@ -98,6 +98,8 @@ def queries_from_config(config_path):
             with open(values) as fp:
                 packfile = fp.read()
                 packcontent = rmcomment.sub('',packfile)
+                print(packcontent)
```
Comment (theopolis) [inline tools/tests/utils.py]: Are these intentional?

Comment (jedi22) [inline tools/tests/utils.py]: My bad

**Comment by osqueryer** [comment]
:no_good: The commit b9764b741110107f812d98f4e0e160956c4be19e (Job results: [1718](https://jenkins.osquery.io/job/osqueryPullRequestAudit/1718/)) failed the code audit or documentation test.

**Pull request merged**